core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -83,7 +83,7 @@ private[spark] class TaskSetManager(
   val copiesRunning = new Array[Int](numTasks)
   val successful = new Array[Boolean](numTasks)
   private val numFailures = new Array[Int](numTasks)
-  // key is taskId, value is a Map of executor id to when it failed
+  // key is taskId (aka TaskInfo.index), value is a Map of executor id to when it failed
```

Comment:
here (and `executorIsBlacklisted`) is what got me confused about the meaning of these identifiers.  I knew it was defintiely *not* the globally unique taskId, aka TID.